packages/ra-data-graphql/src/index.js

```diff
@@ -0,0 +1,126 @@
+import merge from 'lodash.merge';
+import get from 'lodash.get';
+import { ApolloClient } from 'apollo-client';
+import pluralize from 'pluralize';
+import {
+    GET_LIST,
+    GET_ONE,
+    GET_MANY,
+    GET_MANY_REFERENCE,
+    CREATE,
+    UPDATE,
+    DELETE,
+} from 'react-admin';
+
+import buildApolloClient from './buildApolloClient';
```

Comment:
`buildApolloProvider`

Reply:
Nope, it builds an Apollo client which will be used by the data provider